src/hardware/serialport/misc_util.cpp

```diff
@@ -225,18 +227,18 @@ void TCPClientSocket::FlushBuffer()
 	}
 }
 
-void TCPClientSocket::SetSendBufferSize(Bitu bufsize)
+void TCPClientSocket::SetSendBufferSize(const size_t n)
 {
 	// Only resize the buffer if needed
-	if (!sendbuffer || sendbuffersize != bufsize) {
+	if (!sendbuffer || sendbuffersize != n) {
 		delete [] sendbuffer;
-		sendbuffer = new Bit8u[bufsize];
-		sendbuffersize = bufsize;
+		sendbuffer = new uint8_t[n];
+		sendbuffersize = n;
 	}
 	sendbufferindex = 0;
```

Comment:
Renames `bufsize` argument to `n`, if/when we move the the sendbuffer to an STL container.